ui/apps/platform/src/types/networkBaseline.proto.ts

```diff
@@ -0,0 +1,45 @@
+import { L4Protocol, NetworkEntity } from './networkFlow.proto';
```

Comment:
I see this is a new file but I don't see any other files using these values yet. Is it something we will eventually use?

Reply:
Good eyes. Whenever we circle back to Network Graph, it might be useful. For example:

https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Network/networkTypes.ts#L28-L32

```diff
 export type Peer = {
     entity: Entity;
-    portsAndProtocols: {
-        port: string;
-        protocol: Protocol;
-        ingress: boolean;
-    }[];
+    portsAndProtocols: NetworkBaselineConnectionProperties[];
     ingress: boolean;
     egress: boolean;
     state?: ConnectionState;
 };
```